eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -443,6 +443,15 @@
     <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
   </Match>
 
+  <!-- Incorrect flagging, if the response is null a StorageException would have been thrown -->
+  <Match>
+    <Or>
+      <Class name="com.azure.security.keyvault.keys.KeyClient"/>
+    </Or>
+    <Bug pattern="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE"/>
```

Comment:
@JonathanGiles 
Was getting this on all the T return type methods.
Checked with @g2vinay also on this, he said we ideally shouldn't be getting anything like this.
Added this as a temporary fix, would you know more about it.

Reply:
Let's sync up sometime to discuss exactly where this issue is arising and how we should resolve.

Reply:
Keeping this fix, for now.
 Opened an [issue](https://github.com/azure/azure-sdk-for-java/issues/4556) for future investigation.